### DISCUSSION TITLE
Need to keep refreshing Browser

### DISCUSSION THREAD

**Discussion by dkirrane** [community/community#27038] [Actions]
<p>I’m using Chrome Browser Version 91.0.4472.77 (Official Build) (64-bit)</p>
<p>When I run a GitHub Action I have to refresh the browser with F5 to see updates on the page. Otherwise it seems very slow to refresh.</p>
<p>Is there a way to live reload the page faster?</p>

**Comment by mickeygousset** [comment, marked as answer]
<p>Not that I am aware of. The page will usually refresh, but as you have seen, it can take a few seconds. I usually trigger a manual workflow, and click on the workflow again to refresh the view.</p>
